### ISSUE TITLE
_WD_UpdateDriver - Don't Delete WebDriver unless new Driver exists

### ISSUE THREAD

**Issue by seadoggie01** [Danp2/au3WebDriver#197]
Unfortunately, MS broke the normal upgrade path for WebDrivers today. The current "stable" version has only been made available for Mac users(!?). Because of this, _WD_UpdateDriver detected an update was available, attempted to download the new driver, failed, deleted the original driver, and returned an error.

Currently, this functionality can be seen by using `_WD_UpdateDriver("msedge", @AppDataDir & "\WebDrivers\", True, True)`

Edit: I'm looking into a way to fix this

**Comment by seadoggie01** [comment]
The commit mlipok added fixes the issue, it returns False and sets @error = 13 (Not Found) instead of deleting the WebDriver and causing `$FilesInZip = $oShell.NameSpace($sTempFile).items` to cause an unhandle-able AutoIt due to a known issue in AutoIt.

**Comment by mlipok** [comment]
> causing `$FilesInZip = $oShell.NameSpace($sTempFile).items` to cause an unhandle-able AutoIt due to a known issue in AutoIt.
Take a look here:
[$oNameSpace - fix for unhandled AutoIt error](https://github.com/Danp2/au3WebDriver/pull/198/commits/0a91968b54951055491efc5ab0ec0c8d9f8f11cb)

**Comment by seadoggie01** [comment]
Yes, that would fix that error. The error only occurred because the downloaded file didn't exist, causing `$oShell.NameSpace($sTempFile)` to be null, then calling `.items` on it. So, theoretically, this shouldn't ever happen again, but probably better to prevent other unseen errors in the future :)
